portmidi.go

```diff
@@ -88,5 +88,8 @@ func Time() Timestamp {
 
 // convertToError converts a portmidi error code to a Go error.
 func convertToError(code C.PmError) error {
```

Comment:
Another rationalization is that in the C code the PmError type is an enum where 0 and postivie values are "non-error": https://github.com/philandstuff/portmidi/blob/master/pm_common/portmidi.h#L125

so by virtue of this change it more accurately captures the mapping from the C code to Go's error type.